Products/ZPerFactMods/requestWrapper.py

```diff
@@ -0,0 +1,55 @@
+#
+# Hook into event handlers to call request_init/request_end
+#
+
+import zope.component
+import ZPublisher.pubevents
+import six
+import logging
+
+logger = logging.getLogger('Products.ZPerFactMods.requestWrapper')
+
+
+def call_hook(event, method):
+    """
+    Find a method named `method` in the current context and call it, unless the
+    published object has a title that contains NO_REQWRAP.
+    This is also called for a lot of internal methods like ZMI ressources,
+    which sometimes have to title property. Such errors as well as the
```

Comment:
sometimes have to have a title property

Reply:
Actually, it is "sometimes have *no* title property".